content/en/about/community/partners/index.md

```diff
@@ -32,6 +32,7 @@ Istio benefits from an ecosystem of great partners. Check 'em out and learn how
     {{< company_logo link="https://www.tigera.io" logo="./tigera.svg" alt="Tigera" tagline="Zero Trust Network Security & Continuous Compliance for Kubernetes Platforms" >}}
     {{< company_logo link="https://www.vmware.com" logo="./vmware.svg" alt="VMware" >}}
     {{< company_logo link="https://wso2.com" logo="./wso2.svg" alt="WSO2" tagline="The #1 Open Source API Management and Integration Platform" >}}
+    {{< company_logo link="https://layer5.io/" logo="https://raw.githubusercontent.com/layer5io/layer5/master/assets/images/layer5/layer5-tag-community-white-bg.png" alt="Layer5" tagline="The Service Mesh Community - characterizing and contrasting service mesh projects and related technologies" >}}
```

Comment:
This is similar to #6099

- I notice the partners are listed in alphabetical order. To maintain that order, this would need to move up the list.